src/docs/asciidoc/15-examples.adoc

```diff
@@ -30,6 +30,52 @@ task buildImage(type: DockerBuildImage) {
 }
 ----
 
+==== Interacting With a Container Running in the Background
```

Comment:
The title of this section doesn't seem to indicate an actual interaction with the container. From what I can see in the code example it rather shows a full lifecycle of a container: create, start, stop, remove. I think a different title may make more sense.

Reply:
Sounds reasonable. How about `Creation and Management of a Background Container for Local Development`

Reply:
Thanks, let's start with describing the use case first e.g. with a concrete example (not the code).